components/HomeFeed/HomeFeed.js

```diff
@@ -1,17 +1,65 @@
+import { useRef, useEffect, memo } from "react";
 import { FeedNote } from "../Note/Note";
-import { Stack } from "@mantine/core";
 import { useHomeFeed } from "ndk/hooks/useHomeFeed";
+import { VariableSizeList, areEqual } from "react-window";
+import AutoSizer from "react-virtualized-auto-sizer";
 
 export default function HomeFeed() {
   const feed = useHomeFeed();
+  const notes = feed.filter(
+    (note) => !note.event.tags.find((tag) => tag[0] === "e")
+  );
```

Comment:
This could probably be memoized

Reply:
the component doesn't re-render much, so not sure if it would be noticeable. it only re-renders a few times during the initial load of events.